test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -2665,129 +2668,160 @@ func TestSchedulingGatesPluginEventsToRegister(t *testing.T) {
 	}
 
 	tests := []struct {
```

Comment:
Actually, @kerthcet is also modifying this test #124783

@kerthcet which approach would you rather take?

Reply:
I think no way to register empty events with this PR merged, and I think it's ok because the additional cost is run the lightweight hint function, so I'll modify my PR with another refactor after this done.